actions/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use scorecard's official Docker image as base.
+ARG DOCKER_SHA
+FROM gcr.io/openssf/scorecard@sha256:${DOCKER_SHA} as base
+
+# Build our image and update the root certs.
+FROM debian:9.5-slim
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ca-certificates
+
+# Copy the scorecard binary from the official scorecard image.
+COPY --from=base /scorecard /scorecard
+
+# Copy a test policy for local testing.
+COPY actions/testdata/policy.yml .
+
+# Our entry point.
+ADD actions/entrypoint.sh entrypoint.sh
+RUN chmod +x entrypoint.sh
```

Comment:
You shouldn't need this. It copies over the file permissions from the repository. So just commit the file with right permissions.